core/pom.xml

```diff
@@ -125,12 +125,17 @@
       <artifactId>jetty-servlet</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlets</artifactId>
+      <scope>compile</scope>
+    </dependency>
     <!-- Because we mark jetty as provided and shade it, its dependency
```

Comment:
We are actually just using orbit javax.servlet package, so upgrade to javax.servlet is ok. But not sure what I should write for the comment.